DigitalLearningSolutions.Data.Tests/Services/ActivityServiceTests.cs

```diff
@@ -167,6 +170,72 @@ public void GetFilteredActivity_requests_activity_with_correct_parameters()
                 .MustHaveHappened(1, Times.Exactly);
         }
 
+        [Test]
+        public void GetActivityDataFileForCentre_returns_expected_excel_data()
+        {
+            // given
+            using var expectedWorkbook = new XLWorkbook(
+                TestContext.CurrentContext.TestDirectory + ActivityDataDownloadRelativeFilePath
+            );
+            var expectedActivityResult = new List<ActivityLog>
+            {
+                new ActivityLog
+                {
+                    Completed = true,
+                    Evaluated = false,
+                    Registered = false,
+                    LogDate = DateTime.Parse("2020-12-22"),
+                    LogYear = 2020,
+                    LogQuarter = 4,
+                    LogMonth = 12
+                }
+            };
+            A.CallTo(
+                    () => activityDataService.GetFilteredActivity(
+                        A<int>._,
+                        A<DateTime>._,
+                        A<DateTime>._,
+                        A<int?>._,
+                        A<int?>._,
+                        A<int?>._
+                    )
+                )
+                .Returns(expectedActivityResult);

```

Comment:
Could we maybe pull this into a private method to make the test easier to read - e.g. GivenActivityDataServiceReturnsDataInExampleSheet() 

There is more potential for breaking this down a bit to make it easier to read and the assertion looks very generic so could potentially be used in several places/belong to the helper.